app/scripts/modules/titus/rollup.config.js

```diff
@@ -0,0 +1,7 @@
+import baseRollupConfig from '@spinnaker/scripts/config/rollup.config.base.module';
+
+export default {
+  ...baseRollupConfig,
+  input: 'src/titus.module.ts',
+  external: ['@spinnaker/amazon', '@spinnaker/core', '@spinnaker/docker'],
```

Comment:
the base config uses a plugin to mark everything as external.  What does this do?  Is it additive? Why is it necessary when (shouldn't the plugin do this for us?)

Reply:
Oops, this should be handled by the plugin. I had this in here before the update. Removing it now.